.nvidia-ci.yml

```diff
@@ -433,6 +489,23 @@ release:ngc-rhel8.10:
     - 'echo "Signing the image ${IMAGE_NAME}:${IMAGE_TAG}"'
     - ngc-cli/ngc registry image publish --source ${IMAGE_NAME}:${IMAGE_TAG} ${IMAGE_NAME}:${IMAGE_TAG} --public --discoverable --allow-guest --sign --org nvidia
 
+sign:ngc-precompiled-ubuntu24.04:
+  extends:
+    - .driver-versions-precompiled-ubuntu24.04
+    - .dist-ubuntu22.04
+    - .release-generic
+    - .release:ngc-variables
+    - .sign:ngc
+  variables:
+    DIST: signed_ubuntu24.04
+    BASE_TARGET: noble
+    PRECOMPILED: "true"
+  needs:
+    - release:ngc-precompiled-ubuntu24.04
+  rules:
+    # Only run NGC release job on scheduled pipelines
+    - if: $CI_PIPELINE_SOURCE == "schedule"
+
```

Comment:
@shivakunv this needs to be commented out as well since it is referring to release:ngc-precompiled-ubuntu24.04 which is commented out.

```suggestion
#sign:ngc-precompiled-ubuntu24.04:
#  extends:
#    - .driver-versions-precompiled-ubuntu24.04
#    - .dist-ubuntu22.04
#    - .release-generic
#    - .release:ngc-variables
#    - .sign:ngc
#  variables:
#    DIST: signed_ubuntu24.04
#    BASE_TARGET: noble
#    PRECOMPILED: "true"
#  needs:
#    - release:ngc-precompiled-ubuntu24.04
#  rules:
#    # Only run NGC release job on scheduled pipelines
#    - if: $CI_PIPELINE_SOURCE == "schedule"

```